Harmony/Tools/AccessTools.cs

```diff
@@ -56,6 +56,14 @@ public static Type TypeByName(string name)
 		/// 
 		public static Type[] GetTypesFromAssembly(Assembly assembly)
```

Comment:
As discussed earlier, the best option is to only catch `ReflectionTypeLoadException` and use the valid types for the result. Keep throwing any other exception.

Also refactor any code that uses `assembly.GetTypes()` to use this method.

Reply:
Alright, I'll do this in the followup PR to this (the one that actually does the fixes).